### PR TITLE
Feature: Add namespace to topo and update deps

### PR DESCRIPTION
**Pull request by smilebot**
Rearranges deps to be in order.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ const schema = Joi.object({
```
Comment (tuckbick) [inline lib/index.js]: In the schema above, I believe `Joi.func()` was renamed to `Joi.function()`. Though I suppose there may still be an alias for it...

**Comment by tuckbick** [comment]
Hello again! Is there an ETA on the release of these changes?

**Comment by tlivings** [comment]
Do we know if this is techincally a breaking change to node version support?

**Comment by tuckbick** [comment]
None of the updated dependencies mention anything about dropping support for any version of Node. So - just a fix (or a feature), up to you.

**Pull request merged**